client/src/modules/accounts/accounts.html

```diff
@@ -1,37 +1,51 @@
-<div class="flex-header static">
+<div class="flex-header">
   <div class="bhima-title">
     <ol class="headercrumb">
       <li class="static">{{ "TREE.FINANCE" | translate }}</li>
       <li class="title">{{ "ACCOUNT.TITLE" | translate }}</li>
     </ol>
 
-   <div class="toolbar">
+    <div class="toolbar">
+      <fieldset ng-disabled="AccountsCtrl.transactions.isEditing()">
+        <div class="toolbar-item">
+          <div uib-dropdown dropdown-append-to-body data-action="open-tools">
+            <a class="btn btn-default" uib-dropdown-toggle>
+              <span class="fa fa-bars"></span> <span class="hidden-xs" translate>FORM.LABELS.MENU</span> <span class="caret"></span>
+            </a>
+            <ul uib-dropdown-menu role="menu" class="dropdown-menu-right">
+              <li role="menuitem">
+                <a ng-href="/reports/finance/accounts/chart/?{{ AccountsCtrl.Accounts.download('pdf') }}" download="{{ 'ACCOUNT.TITLE' | translate }}">
+                  <span class="fa fa-file-pdf-o"></span> <span translate>DOWNLOADS.PDF</span>
+                </a>
+              </li>
+              <li role="separator" class="divider"></li>
 
-     <div class="toolbar-item">
-
-       <button
-         class="btn btn-default"
-         ng-class="{'btn-info' : AccountsCtrl.gridOptions.enableFiltering}"
-         data-method="filter" ng-click="AccountsCtrl.toggleInlineFilter()">
-        <i class="fa fa-filter"></i>
-       </button>
-
-       <a class="btn btn-default text-capitalize" data-method="create" ui-sref="accounts.create">
-         <i class="fa fa-plus"></i>
-         <span translate> ACCOUNT.ADD </span>
-       </a>
-       <bh-pdf-print
-        pdf-url="/reports/finance/accounts/chart"
-        options="AccountsCtrl.parameter">
-      </bh-pdf-print>
+              <li role="menuitem">
+                <a ng-href="/reports/finance/accounts/chart/?{{ AccountsCtrl.Accounts.download('csv') }}" download="{{ 'ACCOUNT.TITLE' | translate }}">
```

Comment:
@lomamech, while you are at it, @jeremielodi recently added downloading as Excel (https://github.com/IMA-WorldHealth/bhima-2.X/pull/2106).  Could you add that feature into this list as well?